### DISCUSSION TITLE
Error at running yarn install or yarn add

### DISCUSSION THREAD

**Discussion by felipsbreno** [gatsbyjs/gatsby#37081] [Help] [answered]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

I'm using the `babel-preset-gatsby` in my project, but at running the script `yarn add` occurs a error. The error is about the engine, conform image below.

![image](https://user-images.githubusercontent.com/50807243/203466231-3279f2ea-8d20-4921-9a3e-5be7e8ef05d7.png)

### Reproduction Link

https://www.loom.com/share/362534a2bab24a0db7e0aa6aa77982a8

### Steps to Reproduce

1. running the command `yarn add`
2. running this command with node in version `16` or `17`
3. verify console after running the command
...

### Expected Result

Verify if is possible change the engine to others version of [Node](https://nodejs.org/en/)

### Actual Result

A error is returned, conform image below.
 
![image](https://user-images.githubusercontent.com/50807243/203466231-3279f2ea-8d20-4921-9a3e-5be7e8ef05d7.png)

### Environment

```shell
$ node -v
v16.14.2

$ npm -v
8.5.0

$ yarn -v
1.22.18

"gatsby": "^4.24.8",

 "babel-preset-gatsby": "^3.1.0",
```

**Comment by LekoArts** [comment, marked as answer]
`babel-preset-gatsby` v3 expects Node 18. That's expected behavior. If you want to use the version that was used during the Gatsby 4 lifecycle you need to install `babel-preset-gatsby`v2
